.gitignore

```
@@ -14,6 +14,9 @@ doc/build/*
 doc/_build/*
 freud/*.so
 freud/*.cpp
+freud/*.cxx
+freud/libfreud.dylib
```

Would prefer to match the catch-all style. We shouldn't ever commit a dylib file.
```suggestion
freud/*.dylib
```